packages/react-component-library/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+
+import { render, screen } from '@testing-library/react'
+
+import { color, spacing } from '@royalnavy/design-tokens'
+
+import { ProgressBar } from './ProgressBar'
+
+describe('ProgressBar', () => {
+  it('should display the text percentage', () => {
+    render(<ProgressBar value={50} showPercentage />)
+    expect(screen.getByTestId('percentage-text')).toHaveTextContent('50%')
+  })
+
+  it('should render the simple component', () => {
```

Comment:
This test is not really required because the assertions are implicit in other tests. eg there are many `screen.getByRole('progressbar')` throughout.